scania-digital-design-system/sdds
# Task 1696

**Describe pull-request**  
Data table as web component

**Solving issue**  
Fixes: 
[AB#1696](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1696)
[AB#1787](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1787)
[AB#1780](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1780)

**How to test**  
To be populated after converting from draft to regular PR.
1. Go to...
2. Check in...
3. Run ...

**Screenshots**  
_If applicable, add screenshots to help explain_

**Additional context**  
_Add any other context about the pull-request here._
